pinot-core/src/test/java/org/apache/pinot/core/realtime/impl/fakestream/FakeStreamMessageDecoder.java

```diff
@@ -35,11 +36,13 @@ public void init(Map<String, String> props, Set<String> fieldsToRead, String top
 
   @Override
   public GenericRow decode(byte[] payload, GenericRow destination) {
-    return null;
+    GenericRow row = Fixtures.createSingleRow(((Long) System.currentTimeMillis()).intValue());
```

Comment:
(minor) Consider changing `Fixtures.createSingleRow()` to take `long randomSeed` because `Random` actually takes long seed